app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+  skip_before_action :require_login, only: [:activate, :confirm]
+  layout "login", only: [:activate, :confirm]
+
+  def index
+    @user = User.new
+    @users = User.all
+  end
+
+  def create
+    @user = User.new(user_params)
+    @user.save
+  end
+
+  def activate
```

Comment:
I feel like we should move this to its own controller with REST naming, but I'm not sure on the naming right now.